inc/classes/admin/settings/class-settings.php

```diff
@@ -37,6 +37,16 @@ class Settings {
 	 */
 	private $hidden_settings;
 
+	/**
+	 * Host domain.
+	 *
+	 * @since 3.6
+	 * @see   $this->get_host()
+	 *
+	 * @var string|bool
+	 */
+	private $host;
```

Comment:
We've touched this class. Let's move it to our new architecture, rename, and deprecate this class file.

Reply:
Same file location issue than https://github.com/wp-media/wp-rocket/pull/2554#issuecomment-617475684. Let’s decide what to do.

Reply:
`WP_Rocket\Engine\Settings\Settings`

Reply:
Done in https://github.com/wp-media/wp-rocket/pull/2555/commits/908182cf191f3dcef00e78e50f59212c7293fb00.